doc/testing.md

```diff
@@ -1,30 +0,0 @@
-# Testing
-
-(This page is about testing Chestnut itself, *not* about how to do testing in your Chestnut app.)
```

Comment:
Oops this is a different kind of testing, this file should stay around for now. Even though it'll be automated soon™